css/style.css

```diff
@@ -1,47 +1,54 @@
-/* body {
-	max-width: 940px;
-	margin: 0 auto;
-} */
+body {
+	font-family: Georgia, 'Times New Roman', Times, serif;
+}
 a {
 	text-decoration: none;
 }
+header {
+	display: flex;
+	justify-content: center;
+}
 header div{
 	display: flex;
+	justify-content: space-between;
+	align-items: center;
 	width: 913.5px;
 	height: 137.3px;
```

Comment:
Don't be so accurate, it is more then pixel perfect. I haven't tried this, but you can check on this https://habr.com/ru/post/195414/.